crypto/fipsmodule/rand/urandom.c

```diff
@@ -78,6 +78,11 @@
 #endif
 #endif
 
+#if defined(OPENSSL_OPENBSD)
+#define URANDOM_BLOCKS_FOR_ENTROPY
+// OpenBSD does not support getrandom, so we will fallback to /dev/urandom
```

Comment:
OpenBSD supports [`getentropy()`](https://man.openbsd.org/getentropy), which is supported and preferred in upstream BoringSSL (see: [boringssl@53f09ad241ddf7e37a426a5e3c6dfa6d4dca9706](https://boringssl.googlesource.com/boringssl/+/53f09ad241ddf7e37a426a5e3c6dfa6d4dca9706)), so it looks that it was lost in a merge?